navapbc/labs-decision-support-tool
# DST-401: inline citations

## Ticket

https://navalabs.atlassian.net/browse/DST-401


## Changes
> add inline footnotes for citations

## Testing

<img width="753" alt="Screenshot 2024-08-23 at 5 10 17 PM" src="https://github.com/user-attachments/assets/c90c7e36-1f2f-4bb5-8ca3-4780fb90c49b">

1. Navigate to http://localhost:8000/chat/?engine=bridges-eligibility-manual
2. Adjust min citation params to -1
3. Add query, citation should appear
